.github/workflows/conversion_script.yml

```diff
@@ -0,0 +1,52 @@
+name: Generate Conversion Script
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*.csv.gz'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/*_diff.txt'
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/all_ontology.json.gz'
+
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-conversion-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.11
+      - name: Python cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          pip install -r migration_assistant/requirements.txt
+      - name: Run Conversion Script Generator
+        run: |
+          make generate-conversion-script
+      - name: Get Version
+        id: get_version
+        run: |
+          echo "version=$(make show-current-version)" >> $GITHUB_ENV
```

Comment:
This will return the schema version before the version bump. We should either:
1) trigger this workflow after the release to PyPI (which contains the step that bumps the version), 
2) bump the version earlier in the workflow in its own step so it's picked up both here and in the PyPI release step, or 
3) Change the PR message to read 'update convert.py from <env.version> to <env.version + 0.1.0>'